concent_api/core/validation.py

```diff
@@ -262,6 +264,11 @@ def get_validated_client_public_key_from_client_message(golem_message: message.b
             )
 
     elif isinstance(golem_message, message.tasks.TaskMessage):
+        if not golem_message.is_valid():
+            raise Http400(
```

Comment:
Please do not use `Http400` in validation functions. As discussed earlier, validation layer should not be tied to any particulat transportation layer. If there is no suitable exception defined, please define a new one and map it to proper HttpResponse/JsonResponse in decorator layer.

Reply:
I've made custom `GolemMessageValidationError` for this. Should I apply it to all `Http400` exceptions in this function?